src/main/webapp/populateData.js

```diff
@@ -89,3 +89,35 @@ function populateTasks() {
       });
 }
 
+/**
+ * Populate Calendar container with user's events
+ */
+function populateCalendar() {
```

Comment:
Suggestion for maybe another PR: `populateGmail()`, `populateTasks()` and `populateCalendar()` look very oddly identical to me. It would be nice if we could refactor and write a helper function to do both of these.

Reply:
Just created an issue #68